content/devdocs/references/p2p_networking.md

```diff
@@ -136,7 +197,15 @@ The following annotated hexdump shows a `GETB` message.  (The
 message header has been omitted.)
 
 {{ <highlight text> }}
-TODO : GETB message hex dump
+GetBlocksMessage:
+```
+14 00 00 00 00 00 00 00 d2 04 00 00 00 00 00 00 
+2e 16 00 00 00 00 00 00 ........................... Full message
```

Comment:
No need to add full message . It's too much , especially if messages are long. Annotated message would be enough. Please remove